Services/HandlerManager.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+
+namespace Symfony\Bundle\MonologBundle\Services;
+
+
+use Monolog\Handler\HandlerInterface;
+
+class HandlerManager
```

Comment:
I don't like `*Manager` class name. Could you rename it to `HandlerHolder`, or something similar ?

Reply:
Yes, agree that `Manager` is not very decriptive, not sure `Holder` is any more descriptive though. Have changed it to `HandlerLifecycleManager` as that feels closer to what it actually does. Is that OK, or do you have other ideas?

Reply:
Very good name 👍🏼